### ISSUE TITLE
Email: WCAG 3.0 public comment (13 - scoring approach)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#436]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0010.html): 
from [spreadsheet](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0010/20210226drempelvrij_Foundation_-_comments_on_working_drafts_WCAG_3.0.xlsx)

7.1 We are interested in your   feedback on this approach to testing and scoring . Does this approach help large organizations conform even if   their site is not 100% perfect? Do you think that organizations will   interpret that they only need 95% of text alternatives for images and then   stop adding alternative text? Are the bands of numbers for the different   ratings correct? Do people with disabilities in particular feel that this   approach will meet their needs?

This   could work as long as the method is unambiguous, and every tester comes to   the same conclusion. Also there should be a drive for organizations to make   their websites as accessible as possible, by giving them some kind of   platform. And there should be a simple way built into the report for people   (with disabilities) for who the website is not accessible to address the   problem so that the owner can rectify this situation or offer an alternative.   This gives the owner an incentive to comply as fully as possible since it   reduces the costs for a helpdesk of chatbox.

**Comment by cwadamsoforacle** [comment]
Proposed response:
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
